app/cdap/components/WrangleHome/Components/OngoingDataExploration/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { makeStyles } from '@material-ui/core';
+
+const ExplorationCardStyles = makeStyles({
+  gridContainer: {
+    width: '1204px',
+    height: '77px',
+    border: '1px solid #DADCE0',
+    borderRadius: '4px',
+    marginBottom: '10px',
+    cursor: 'pointer',
+  },
+  elementStyle: {
+    width: '301px',
+    flex: 1,
+    paddingTop: '14px',
+    paddingLeft: '23px',
+    paddingBottom: '13px',
+    paddingRight: '23px',
+    display: 'flex',
+
+    '& .MuiTypography-body1': {
+      margin: 'auto 0px',
+      fontSize: '16px',
+      lineHeight: '24px',
+      width: '254px',
+      textOverflow: 'ellipsis',
+      fontWeight: 400,
+      overflow: 'hidden',
+      whiteSpace: 'nowrap',
+      color: '#000000',
+    },
+  },
+  iconStyle: {
+    marginRight: '32px',
+    height: '50px',
+    width: '50px',
+  },
+  percentageStyleRed: {
+    lineHeight: '30px',
+    fontSize: '36px',
+    letterSpacing: '0.15px',
+    color: '#EA4335',
```

Comment:
Please review the colors in [our palette](https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/styles/colors.scss) and try to find a match. (Access them via the [colors TS file](https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/ThemeWrapper/colors.ts).)

Reply:
We dont have the same colors. UI updated with the colors provided by the Design Team